media-api/app/lib/elasticsearch/SearchFilters.scala

```diff
@@ -90,4 +79,9 @@ trait SearchFilters extends ImageFields {
     "pool"
   )
 
+  def filterOrFilter(filter: Option[FilterBuilder], orFilter: Option[FilterBuilder]): Option[FilterBuilder] = (filter, orFilter) match {
```

Comment:
@theefer I can only think of:
```scala
import scalaz.syntax.apply._, scalaz.std.option._

^(filter, orFilter)(filters.or) orElse filter orElse orFilter
```

You might one day want to generalise it to any number of filters:

```scala
List(filter, orFilter, anotherFilter).flatten.reduceLeft(filters.or)
```